client/components/FormattingBarLegacy/Media/media.scss

```diff
@@ -6,7 +6,7 @@
 	.options {
 		flex: 0 0 auto;
 		border-right: 1px solid #ddd;
-		overflow: scroll;
+		overflow: auto;
```

Comment:
This thing tends to scroll regardless

![Screen Shot 2020-07-30 at 2 01 33 PM](https://user-images.githubusercontent.com/2208769/88959413-f6ada780-d26f-11ea-8589-e3bb92e6e166.png)